docs/pager.md

```diff
@@ -0,0 +1,89 @@
+# Pager composable
+
+[![Maven Central](https://img.shields.io/maven-central/v/dev.chrisbanes.accompanist/accompanist-pager)](https://search.maven.org/search?q=g:dev.chrisbanes.accompanist)
+
+A library which provides a horizontally paging layout for Jetpack Compose. If you've used Android's [`ViewPager`](https://developer.android.com/reference/kotlin/androidx/viewpager/widget/ViewPager) before, it has similar properties.
```

Comment:
Will this work with vertical paging layouts (i.e. tiktok feed) as well?

Reply:
Not initially, but we should bake in the fact that this is a horizontal pager into the API